python/src/scipp/plotting/model.py

```diff
@@ -83,6 +80,29 @@ def dims(self, dims):
         self._dims = dims
         self._dims_updated()
 
+    def _setup_coords(self, array):
+        data = array.data
+        if array.bins is not None:
+            data = self._setup_event_coords(data)
+        coord_list = {dim: self._axis_coord(array, dim) for dim in array.dims}
+        return sc.DataArray(data=data, coords=coord_list, masks=to_dict(array.masks))
+
+    def _setup_event_coords(self, data):
+        tmp = data.bins.constituents
+        coord_list = {}
+        array = tmp['data']
+        for dim in data.dims:
+            # Drop any coords not required for resampling to save memory and compute
+            if dim in array.meta:
+                coord = array.meta[dim]
+                if typing.has_datetime_type(coord):
+                    coord = coord - coord.min()
```

Comment:
Why subtract the min?

Reply:
This is how datetime formatting/rebinning is handled, nothing new here, just doing the same that is done for dense coords.

Reply:
Basically we cannot rebin datetime, so the min is subtracted such that the integer-valued time-difference can be rebinned instead.

Reply:
Thanks that makes sense.